framework/tests/all-systems/java8/memberref/MemberReferences.java

```diff
@@ -31,7 +31,7 @@ Object func1(Object o) {
         void context() {
             FunctionMR<Object, Object> f1 = super::func1;
             // TODO: Issue 802: type argument inference
-            //:: warning: (methodref.inference.unimplemented)
+            // TODO: should expect warning: (methodref.inference.unimplemented)
```

Comment:
@smillst It's not clear to us why the move of javac from the bootclasspath to the normal classpath had an effect on this single expected error.
Do you understand this change in behavior? Do you mind if we disable this warning for now?

Reply:
Disabling is fine.  I don't think the code that issues that warning is correct.  See https://github.com/typetools/checker-framework/issues/1319.